components/camel-netty-http/src/main/java/org/apache/camel/component/netty/http/NettyHttpProducer.java

```diff
@@ -51,6 +53,10 @@ public NettyHttpProducer(NettyHttpEndpoint nettyEndpoint, NettyConfiguration con
 
     @Override
     protected void doInit() throws Exception {
+        if (isProxyProtocol()) {
```

Comment:
I wonder if this can this be checked in createProducer instead?

Also we do not log and throw exception, only throw exception

Reply:
> I wonder if this can this be checked in createProducer instead?
yeah, it's better there. moved.
> Also we do not log and throw exception, only throw exception
ok, removed the log.